dbt/include/spark/macros/materializations/seed.sql

```diff
@@ -69,6 +69,7 @@
     {{ clustered_cols(label="clustered by") }}
     {{ location_clause() }}
     {{ comment_clause() }}
+    {{ dbt_spark_tblproperties_clause() }}
```

Comment:
@etheleon : Could you rename the `dbt_spark_tblproperties_clause` to `spark__tblproperties_clause` and create a similar pattern as for the other clauses?


```
{% macro tblproperties_clause() %}
  {{ return(adapter.dispatch('tblproperties_clause', 'dbt')()) }}
{%- endmacro -%}

{% macro spark__tblproperties_clause() %}
...
{%- endmacro -%}

```

Reply:
sure!

Reply:
sure, I've submitted my commit

Reply:
@JCZuurmond if you have time for this review